src/accidental.ts

```diff
@@ -15,38 +15,58 @@ import { Flow } from './tables';
 import { Music } from './music';
 import { Modifier } from './modifier';
 import { Glyph } from './glyph';
+import { StaveNote } from './stavenote';
+import { AccidentalCode, AccidentalListItem, AccidentalRenderOptions, AccidentalState, Line } from './types/common';
+import { Voice } from './voice';
+import { Note } from './note';
 
 // To enable logging for this class. Set `Vex.Flow.Accidental.DEBUG` to `true`.
-function L(...args) {
+function L(...args: unknown[]) {
   if (Accidental.DEBUG) Vex.L('Vex.Flow.Accidental', args);
 }
 
-const getGlyphWidth = (glyph) => glyph.getMetrics().width;
+const getGlyphWidth = (glyph: Glyph | null | undefined) => {
```

Comment:
I suggest to go for `undefined` and leave `null` out. You can then simply use `glyph?: Glyph`